src/libraries/Microsoft.PowerFx.Core/Binding/Binder.cs

```diff
@@ -121,7 +121,7 @@ internal sealed partial class TexlBinding
 
         public bool HasLocalScopeReferences { get; private set; }
 
-        public ErrorContainer ErrorContainer { get; } = new ErrorContainer();
+        public ErrorContainer ErrorContainer { get; set; } = new ErrorContainer();
```

Comment:
@bryancroteau-MSFT  - fyi , adding setter

Reply:
Shouldn't this be set in the constructor?

Reply:
@bryancroteau-MSFT Not sure if there's a reason to prefer one pattern over the other, I wanted to limit this PR to purely changes needed for my functionality (and testing)